STANDARDS.md

```diff
@@ -2,65 +2,69 @@
 - A github submitter username
-- The name of the person who selected/sourced the molecules (NEW)
-- A description of the meaning of the entry/molecule keys/names (NEW)
 - Each entry has canonical isomeric explicit hydrogen mapped SMILES
-- Provenance info of CMILES generation
+
+## Each dataset subdirectory in QCA Dataset submission must have the following information:
+- Full environment used to generate dataset
+- Dataset generation script(s) including the following:
+    - How conformers were generated
+    - Origin of SMILES strings
+    - Enumerated stereo flag (True/False)
+    - Enumerated tautomers flag (True/False)
+    - Enumeration keywords/settings
+    - Code used to set up the dataset
+
+ - A descriptive README file, as described below
+
+## Each dataset README must contain the following information:
+
+- All information specified in the dataset description/metadata
+- A changelog (if there are changes, see the revision section below)
+- The name of the person who selected/sourced the molecules 
+- A description of the meaning of the entry/molecule keys/names, if not generated automatically by QCSubmit
 - A set of elements that the dataset contains
-- A set of charges that the dataset contains (NEW)
-- The mean and max molecular weight of molecules the dataset contains (NEW)
-- Enumerated stereo flag (True/False) (NEW)
-- Enumerated tautomers flag (True/False) (NEW)
-- Enumeration provenance info (NEW)
-- Computation blacklist (known failures) (NEW)
-- Dataset status flags (NEW):
-	- Complete: `COMPLETE` `INCOMPLETE`
-	- State: `DONE` `WORKING` `PAUSED`
-	- Compliance: `V3` `NONE`
-	- COMPLETE/DONE/V3; all molecules were successful
-	- INCOMPLETE/DONE/V3; some molecules were not calculated successfully, and won't be retried
-	- INCOMPLETE/WORKING/V3; in progress
-	- INCOMPLETE/DONE/NONE; the dataset does not conform to the standards, and can't be fixed
-	- INCOMPLETE/WORKING/NONE; the dataset does not conform to the standards, but is working anyways
-	- INCOMPLETE/PAUSED/NONE; the dataset does not conform to the standards, and calculations have been suspended
-
-## Each dataset README must contain the following information
-
-All information specified in the dataset
+- A set of charges that the dataset contains
+- The min, mean and max molecular weight of molecules the dataset contains 
 
 ## Each revision must use the following procedure:
 
 A revision means creating a new dataset based on an existing one, with the intent of fixing/improving it.
 
-- The changelog must be copied from the current version, and a new entry added in the new version
-- The dataset version information is updated
+- The changelog must be copied from the current version (if it exists), and a new entry added for the new version. The changelog should describe any changes between the new version and previous version, including QCAIDs of any records that were removed or modified, with an explanation as to why they were removed.
```

Comment:
```suggestion
- The changelog must be copied from the current version (if it exists), and a new entry added for the new version. The changelog should describe any changes between the new version and previous version, including QCArchive record ids of any records that were removed or modified, with an explanation as to why they were removed.
```

Just suggesting to explicitly refer to QCA record IDs instead of by shorthand :-)